GitUI/CommandsDialogs/FormCommit.Designer.cs

```diff
@@ -1162,12 +1166,24 @@ private void InitializeComponent()
             this.Amend.UseVisualStyleBackColor = true;
             this.Amend.CheckedChanged += new System.EventHandler(this.Amend_CheckedChanged);
             // 
+            // ResetAuthor
+            // 
+            this.ResetAuthor.AutoSize = true;
+            this.ResetAuthor.Location = new System.Drawing.Point(0, 0);
+            this.ResetAuthor.Margin = new System.Windows.Forms.Padding(0, 3, 0, 3);
+            this.ResetAuthor.Name = "ResetAuthor";
+            this.ResetAuthor.Size = new System.Drawing.Size(97, 17);
+            this.ResetAuthor.TabIndex = 0;
+            this.ResetAuthor.Text = "&Reset Author";
```

Comment:
Please avoid conflict with `this.Reset` "&Reset all changes":
```suggestion
            this.ResetAuthor.Text = "R&eset Author";
```